14/umbraco-cms/reference/security/two-factor-authentication.md

```diff
@@ -253,15 +254,15 @@ public class UmbracoUserAppAuthenticator : ITwoFactorProvider
     /// <param name="userOrMemberKey">The key of the user or member</param>
     /// <param name="secret">The secret that ensures only this user can connect to the authenticator app</param>
     /// <returns>The required data to setup the authenticator app</returns>
-    public Task<object> GetSetupDataAsync(Guid userOrMemberKey, string secret)
+    public Task<ISetupTwoFactorModel> GetSetupDataAsync(Guid userOrMemberKey, string secret)
     {
         IUser? user = _userService.GetByKey(userOrMemberKey);
```

Comment:
Getting a warning about this, is it ok to leave this as it is?
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/2b828b36-9c7c-4a69-8219-0c36404f139f)